### ISSUE TITLE
Vocalizer spells wortds in the NVDA menu instead of saying them again.

### ISSUE THREAD

**Issue by mikebayus** [nvaccess/nvda#15940]
### Steps to reproduce:
Download a Vocalizer expressive TTS voice, go to the NVDA menu by pressing NVDA key plus N, press down arrow once and the vocalizer voice will spell the first word.

### Actual behavior:

The vocalizer voices will spell the first word and not just say it.
### Expected behavior:

NVDA should say "prefferences"
#### NVDA version:
NVDA v2024.1 beta1
#### Windows version:
Windows 11 latest build
#### Name and version of other software in use when reproducing the issue:
NA
#### Other information about your system:
NA
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA v2023.3 doesn't do this.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, I am using Vocalizer Expressive for SAPI5 Screen readers.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I don't know, I never tried that.

**Comment by CyrilleB79** [comment]
Hello @mikebayus 

What happens if you uncheck "Use spelling functionality if supported" in NVDA speech settings?

**Comment by mikebayus** [comment]
Vocalizer expressive voices say instead of spell.

Great workaround.

**Comment by CyrilleB79** [comment]
OK. So that's probably the same issue as the one with Code Factory add-on (see #14900 and [this thread about MathCAT](https://nvda.groups.io/g/nvda/topic/102523473#112998)).

Their TTS do not seem to support correctly commands to spell characters when they are embedded in a broader speech sequence, either with the add-on or with the SAPI5 interface.

You'd probably report the issue to them, e.g. with a link to this ticket. Unfortunately, it seem that they do not answer very quickly (according to [this thread](https://nvda.groups.io/g/nvda/topic/103075298#113719)).

**Comment by mikebayus** [comment]
I believe it's not that they don't answer very quickly, I think they have
no idea what we are talking about.

My other workaround is to default my NVDA Screen Reader to one of my
CereProc voices.

I have the Piper TTS addon, or rather I had, until this beta version of
NVDA broke it.

It's the same with Acapela until that addon gets updated as well.

I really really really really like the Piper TTS addon, and I am going to
go check to see if there is an update just as soon as I send this.

Best,
Mike

**Comment by seanbudd** [comment]
Closing - this is an issue with the add-on. Please report it to the add-on authors.

**Comment by mikebayus** [comment]
You might note that I said earlier in this thread that I don't have the
Vocalizer for NVDA Add-on, The Vocalizer expressive driver I have is a
SAPI5 Synth.

Coad Factory is non supportive and has no idea what I am talking about, and
has no desire to help.

This issue of which I speak was fixed in 2023.2 and is broken again in
2023.1 Beta 2.

Actually, it is 2024.1 Beta 1 but I have since updated.

So if you devs want to close this issue all well and good.

I'll just either default to another synth that works, or unchecked the use
spelling functionallity checkbox as a work around.

Happy new year.

Cheers.
